packages/connectkit/src/components/Pages/Solana/ConnectSolana/index.tsx

```diff
@@ -59,15 +121,42 @@ const ConnectSolana: React.FC = () => {
         </ModalContent>
       )}
 
+      {isOnIOS && solanaWallets.wallets.length === 0 && (
+        <ModalContent
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            gap: 16,
+          }}
+        >
+          <ModalH1>Open this page and complete in your wallet</ModalH1>
```

Comment:
after we ship, hallway test this (= watch people encounter it for first time on their phone) and see if they understand